### ISSUE TITLE
Parent module not loaded problem.

### ISSUE THREAD

**Issue by SoulFireMage** [dfelinto/virtual_reality_viewport#29]
![bug](https://cloud.githubusercontent.com/assets/2396080/26037201/8ef29c34-38e6-11e7-9fa3-c5bed70da508.jpg)
I managed to look at one item in blender but now I get this problem. 
 
Basically it just whinges that parent module is not loaded, cannot perform relative import.

Thanks for any advice!

**Comment by dfelinto** [comment]
Hi,

How did you download the add-on? Did you get it from the link mentioned in
the README file (correct way)?

Or did you just used github option to download repository as .zip (wrong
way)?

On May 14, 2017 21:47, "Richard Griffiths" <notifications@github.com> wrote:

I managed to look at one item in blender but now I get this problem.

As there wasn't anywhere convenient to just upload the screenshot on
Github, I've quickly dropped to an image hosting site.

Basically it just whinges that parent module is not loaded, cannot perform
relative import.

Thanks for any advice!

https://ibb.co/krkfo5

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/dfelinto/virtual_reality_viewport/issues/29>, or mute
the thread

**Comment by SoulFireMage** [comment]
No, I used the zip in the link, used it a few times now. 

The temporary workaround is quite simply, delete the plugin from the Roaming folder in appdata and remove it from blender entirely. Then reinstall from the zip file.

This seems to correct the problem.

Also it appears if the VR is running when you go to open a new Blender file, you'll get this error as it doesn't handle the fact you've opened a different type of UI window. This causes the unload crash I think - the only recovery possible being the full reinstall of the plugin.

Hope this helps someone else.
